s/nub-context/nub-context.ts

```diff
@@ -0,0 +1,70 @@
+import {dispatchCustomEvent} from "../nub-keyboard/utils/dispatchCustomEvent.js"
+import {keys} from "../nub-keyboard/utils/keys.js"
+import {styles} from "./style.css.js"
+
+export class NubContext extends HTMLElement {
+	shadow = this.attachShadow({
+		mode: "open",
+		delegatesFocus: false,
+	})
+
+	constructor() {
+		super()
+		let editable = false
+		this.shadow.innerHTML = `
+		<div>
+		<button class="flip">flip mode</button>
+		<slot>
+		</slot>
+		</div>
+		`
+		const style = document.createElement("style")
+		style.textContent = styles
+		this.shadowRoot?.append(style)
+		const flipButton = this.shadowRoot?.querySelector<HTMLElement>(".flip")
+		const nubContext = document.querySelector("nub-context")
+		nubContext?.addEventListener("nub-input", (e: any) => {
```

Comment:
`nub-input` listener should be applied to `this`

and i think i might prefer underscores for event names like `nub_input`... maybe? :shrug: